### PR TITLE
[NEW] Sort List, View Mode and Group by revamp

### PR DESCRIPTION
**Pull request by ggazzo**
image

### CODE REVIEW

File: app/ui-sidenav/client/SortList.js [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+const style = {
+	textTransform: 'uppercase',
+};
+export function SortList() {
+	return <><div className='rc-popover__column'>
+		<SortModeList/>
+		<ViewModeList/>
+		<GroupingList/>
+	</div>
+	</>;
```
Comment (tassoevan) [inline app/ui-sidenav/client/SortList.js:39]:
```suggestion
	return <>
		<div className='rc-popover__column'>
			<SortModeList/>
			<ViewModeList/>
			<GroupingList/>
		</div>
	</>;
```

**Comment by zdumitru** [comment]
The icon "AZ with an arrow" doesn't reflect what the menu actually does. It needs a new icon.

**Comment by kaiiiiiiiii** [comment]
And "Group Discussions" is still visible, even though discussions are disabled ... but like many times before, no one cared about my PR comment ...

**Comment by ggazzo** [comment]
@kaiiiiiiiii sorry but I cant see your previous comment where is the comment? that one is the first... did you finished your "review"?

**Pull request merged**